packages/cli/src/command-utils/common-options.ts

```diff
@@ -76,6 +76,11 @@ export const OPTIONS = {
     description: "Pass the disable remote fonts flag into chromium",
     default: false,
   },
+  noSandbox: {
+    boolean: true,
+    description: "Pass the no sandbox flag into chromium",
+    default: false,
+  },
```

Comment:
If we start passing a lot of these we may want some more generic `--chromiumFlags` option.

Reply:
But I think good like this for now. It does reduce the API surface / otherwise we wouldn't know what flags people may be overriding.

Reply:
Yes, I also thought about doing that if we need to control many flags.

Reply:
The question being how to properly handle the common options and potential overrides.